src/Mod.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Laminas\Filter;
+
+use Laminas\Filter\AbstractFilter;
+use Laminas\Filter\Exception\InvalidArgumentException;
+
+class Mod extends AbstractArithmeticOperation
+{
+    /**
+     *
+     * {@inheritdoc}
+     * @see \Laminas\Filter\FilterInterface::filter()
+     */
+    public function filter($value)
+    {
+        if (! isset($this->options['operand'])) {
+            throw new InvalidArgumentException(sprintf('%s expects argument value; received none', __METHOD__));
+        }
+
+        $value = (float) $value;
+        $operand = (float) $this->options['operand'];
+        return ($value % $operand);
```

Comment:
This doesn't do what you think it does. From the manual:

> Operands of modulo are converted to int before processing. For floating-point modulo, see fmod().

In other words, using the `%` operator will always return an integer. If you want to work with floats, you need to use `fmod()`. So, perhaps two variants, `Mod` and `FloatMod`, are needed.